render_macros/src/element_attribute.rs

```diff
@@ -94,14 +94,41 @@ impl Hash for ElementAttribute {
 
 impl Parse for ElementAttribute {
     fn parse(input: ParseStream) -> Result<Self> {
-        let name = AttributeKey::parse_separated_nonempty_with(input, syn::Ident::parse_any)?;
-        let not_punned = input.peek(syn::Token![=]);
+        let mut name: syn::punctuated::Punctuated<proc_macro2::Ident, proc_macro2::Punct> =
+            syn::punctuated::Punctuated::new();
+
+        // Parse the input up to the space
+        loop {
+            let value = syn::Ident::parse_any(&input).unwrap();
+            name.push_value(value);
+
+            if input.peek(syn::Token![=]) {
+                break;
+            }
+
+            let punct = input.parse().unwrap();
+            name.push_punct(punct);
+        }
+
+        // Peak for incoming equals to check if its punned
+        let mut not_punned = input.peek(syn::Token![=]);
+
+        if !not_punned {
+            not_punned = input.peek2(syn::Token![=]);
+        }
+
+        if !not_punned {
+            not_punned = input.peek3(syn::Token![=]);
+        }
```

Comment:
Because there are multiple sets of punctuation that can occur in a row we look forward to see if any of the next 3 `Punctuated`'s have an equals in it. This _may_ ultimately become a limitation of the library that it can only handle 2 distinct piece of punctuation before the equals though that would be a vast improvement.